src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.scss

```diff
@@ -2,193 +2,199 @@
+  width: $width100;
+  float: left;
+  .link-header{
+    display: flex;
+    background: $color-pf-black-200;
+    align-items: center;
+    .content__title{
+      cursor: pointer;
+      padding: em(22) em(5);
+    }
+    .selected-tab {
+      font-weight: 600;
+    }
+    .link-header-nav-list{
+      width: auto;
+      padding: 0;
+      list-style: none;
+      white-space: nowrap;
+      -webkit-overflow-scrolling: touch;
+      overflow-x: auto;
+      padding: em(5);
+      margin: 0;
+      display: -webkit-box;
+      display: -ms-flexbox;
+      display: flex;
+      -ms-flex-flow: row nowrap;
+      flex-flow: row nowrap;
+      .link-header-nav{
+        padding: em(5) em(15);
+        @extend .dib;
+        border-right: 1px solid $color-pf-black-400;
+        text-align: $textCenter;
+        span{
+          float: none;
+          font-size: em(18);
         }
-        .selected-tab {
-          font-weight: 600;
+        .link-header-nav-text{
+          padding: em(5);
+          font-size: em(12);
         }
-        .link-header-nav{
+      }
+    }
+  }
+  .link-content{
+    width: $width100;
+    float: left;
+    background: $color-pf-white;
+    border-bottom: 1px solid $color-pf-black-200;
+    > div{
+      padding: 0;
+    }
+    .link-content-create-link{
+      text-align: $textCenter;
+      width: $width100;
+      float:left;
+      padding: em(20);
+      .add-circle-icon{
+        color: $color-pf-black-400;
+        font-size: em(48);
+      }
+      h2{
+        color: $color-pf-black-500;
+        font-weight: 500;
+        margin: em(5) 0;
+        float: none;
+      }
+      .create-link-btn{
+        padding-left: em(10);
+        padding-right: em(10);
+      }
+    }
+    .link-item-hdr{
+      background: $color-pf-black-100;
+      padding: em(10) em(40);
+      width: $width100;
+      float: left;
+      p{
+        margin: 0 em(20) 0 0 ;
+        padding: 0;
+      }
+    }
+    .link-editor{
+      width:$width100;
+      float: left;
+      padding: em(15) 0 em(5) 0;
+      > div{
+        margin:em(5) 0;
+      }
+      .detail-status-dropdown{
+        position: relative;
+        .caret{
+          margin-top: em(5);
+        }
+        .dropdown-text{
+          text-align: left;
+          float: left;
+        }
+        .link-dropdown {
+          left: em(20);
+          right: em(20);
+          li{
+            padding: 0 em(5);
             cursor: pointer;
-            flex: 1 0 auto;
-            padding: em(5) em(10);
-            float: left;
-            border-right: 1px solid $color-pf-black-400;
-            text-align: center;
-            span{
-                float: left;
-                font-size: em(18);
-            }
-            .link-header-nav-text{
-                padding: em(5);
-                font-size: em(12);
-            }
+          }
         }
+      }
     }
-    .link-content{
+    .link-list-wrap{
+      width: $width100;
+      float: left;
```

Comment:
Discussed: to avoid layout breakage in case of unexpected type of contained items.